2-mandatory/1-weather-report.js

```diff
@@ -12,14 +12,19 @@
 */
 
 function getTemperatureReport(cities) {
-    // TODO
+    const statements = [];
+    cities.forEach(city => {
+        statements.push('The temperature in '.concat(city, ' is ',
```

Comment:
Prefer using string interpolation instead of the `concat` method https://syllabus.codeyourfuture.io/js-core-1/week-1/lesson#string-concatenation